projects/admin/src/app/record/detail-view/entities-detail-view/local/entities-local-global.component.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019-2023 RERO
+ * Copyright (C) 2019-2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'admin-entities-local-global',
+  template: `
+  <dl class="row mb-0">
+  <ng-container *ngIf="record.source_catalog">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
```

Comment:
remove `-sm-` and simply use `col-3` (remove offset, seems not necessary because UI doesn't adapt for sm screen)